pkg/cmd/extension/manager.go

```diff
@@ -172,7 +179,80 @@ func (m *Manager) InstallLocal(dir string) error {
 	return makeSymlink(dir, targetLink)
 }
 
-func (m *Manager) Install(cloneURL string, stdout, stderr io.Writer) error {
+type BinManifest struct {
+	Owner string
+	Name  string
+	Host  string
+	// TODO I may end up not using this; just thinking ahead to local installs
+	Path string
+}
+
+func (m *Manager) InstallBin(client *http.Client, repo ghrepo.Interface) error {
+	var r *release
+	r, err := fetchLatestRelease(client, repo)
+	if err != nil {
+		return err
+	}
+
+	suffix := m.platform()
+	var asset *releaseAsset
+	for _, a := range r.Assets {
+		if strings.HasSuffix(a.Name, suffix) {
+			asset = &a
+			break
+		}
+	}
+
+	if asset == nil {
+		return fmt.Errorf("%s unsupported for %s. Open an issue: `gh issue create -R%s/%s -t'Support %s'`",
```

Comment:
Nit: even though a space after a shorthand flag isn't necessary, I think examples and printed instructions would be better if they included them for readability.